akka-http-aggregator/src/main/scala/com/pagerduty/akka/http/aggregator/AggregatorController.scala

```diff
@@ -15,22 +21,191 @@ trait AggregatorController[AuthConfig <: HeaderAuthConfig, AddressingConfig] {
   val authConfig: AuthConfig
   def headerAuthenticator: HeaderAuthenticator
   implicit def httpProxy: HttpProxy[AddressingConfig]
+  def requestHandler: RequestHandler[HeaderAuthConfig, AuthConfig#AuthData]
+
   implicit def executionContext: ExecutionContext
   implicit def materializer: Materializer
 
-  def prefixAggregatorRoute(
+  def prefixAggregatorRoute[RequestKey, AccumulatedState](
```

Comment:
I really wanted to keep the noise level down in routes files, and I would consider wrapping in `Some` to be just noise. With the variants the user can pretty much just throw in what arguments they have and it will work, assuming they follow the ordering, so hopefully the picking of a variant isn't too onerous? I guess I'm not a huge fan of duplicated wrappers floating around in various places either.

That said, they are kind of a pain to write/maintain. But now that we have them, perhaps we keep them until it becomes obvious that they're getting in the way?